### PR TITLE
Creates Bullet and BulletController class

### PR DESCRIPTION
**Pull request by hareshkh**
After merging this PR Bullet and BulletController class will be created in place and can be called aptly afterwards for the game to shoot bullets whenever necessary.

### CODE REVIEW

File: core/src/com/sdsmdg/kd/gameplay/controllers/BulletController.java [hunk 1]
```diff
@@ -17,22 +17,11 @@ public BulletController(Bullet bullet, Magnus magnus){
     }
 
     public void control(){
-        if (bullet.active && (bullet.x >= Main.screen.x - 5 || bullet.x <= 5 || bullet.y >= Main.screen.y - 5 || bullet.y <= 5)) {
-
+        if (bullet.active && (bullet.x >= (Main.screen.x + (2*bullet.radius)) || bullet.x <= (0-(2*bullet.radius)) || bullet.y >= (Main.screen.y + (2*bullet.radius)) || bullet.y <= (0-(2*bullet.radius)))) {
```
Comment (karandesai-96) [inline core/src/com/sdsmdg/kd/gameplay/controllers/BulletController.java]: Too long line, break it please.

File: core/src/com/sdsmdg/kd/gameplay/controllers/BulletController.java [hunk 2]
```diff
@@ -17,22 +17,11 @@ public BulletController(Bullet bullet, Magnus magnus){
...
             // For preventing glitchy movement at the boundary.
-            if (bullet.x > Main.screen.x) {
-                bullet.x = Main.screen.x;
-            }
-            if (bullet.y > Main.screen.y) {
-                bullet.y = Main.screen.y;
-            }
+            bullet.reset(magnus);
```
Comment (karandesai-96) [inline core/src/com/sdsmdg/kd/gameplay/controllers/BulletController.java]: It is fine for now, as Magnus is not moving, for debugging purposes. But you need to modify `reset()` afterwards, as setting them to Magnus' center would let them stay below it and expose them when Magnus leaves that position. Did you get me ?

File: core/src/com/sdsmdg/kd/gameplay/controllers/BulletController.java [hunk 3]
```diff
@@ -5,21 +5,16 @@
 import com.sdsmdg.kd.gameplay.objects.Magnus;
```
Comment (hareshkh) [inline core/src/com/sdsmdg/kd/gameplay/controllers/BulletController.java]: This line is obsolete now and can be removed.

**Comment by karandesai-96** [comment]
@hareshkh I am signing off this PR. Please check the diffs once and if you find anything untidy, clean it up, else merge it. It is good to go in **develop** branch.

**Pull request merged**